src/firebase/actionInDatabase/addingMessagesFromDatabaseToRoom.ts

```diff
@@ -0,0 +1,17 @@
+import { doc, getDoc } from 'firebase/firestore'
+import { firebaseCollection } from '../../types'
+import { db } from '../firebase'
+
+export const addMessagesFromDatabaseToRoom = async (roomName: string) => {
+  const messages = await getDoc(doc(db, firebaseCollection.rooms, roomName))
+    .then((res) => {
+      return res.data()
```

Comment:
arrow function with one should be without return, also one more time arrow function with one argument should be without parenthesis